packages/client/src/__tests__/dataTriggers.test.ts

```diff
@@ -51,106 +95,341 @@ describe("Cloud functions -> Data triggers ->", () => {
-            .get();
-          // check that only the test customer's attendance's deleted, but not the rest of the data
-          expect(snap.data()).toEqual(baseAttendance);
-        });
+      {
+        name: "update booking: interval with notes",
+        initialBooking: {
+          date: baseSlot.date,
+          interval: Object.keys(baseSlot.intervals)[0],
+        },
+        initialAttendance: {
+          bookedInterval: Object.keys(baseSlot.intervals)[0],
+          attendedInterval: Object.keys(baseSlot.intervals)[0],
+        },
+        update: {
+          date: baseSlot.date,
+          interval: Object.keys(baseSlot.intervals)[1],
+          bookingNotes: "This is a booking note",
+        },
+        wantAttendance: {
+          bookedInterval: Object.keys(baseSlot.intervals)[1],
+          attendedInterval: Object.keys(baseSlot.intervals)[1],
+          bookingNotes: "This is a booking note",
+        },
       },
-      { timeout: 20000 }
-    );
+
+      {
+        name: "update booking: notes only",
+        initialBooking: {
+          date: baseSlot.date,
+          interval: Object.keys(baseSlot.intervals)[0],
+        },
+        initialAttendance: {
+          bookedInterval: Object.keys(baseSlot.intervals)[0],
+          attendedInterval: Object.keys(baseSlot.intervals)[0],
+        },
+        update: {
+          interval: Object.keys(baseSlot.intervals)[0],
+          bookingNotes: "This is a booking note",
+        },
+        wantAttendance: {
+          bookedInterval: Object.keys(baseSlot.intervals)[0],
+          attendedInterval: Object.keys(baseSlot.intervals)[0],
+          bookingNotes: "This is a booking note",
+        },
+      },
+
+      {
+        name: "remove booking notes",
+        initialBooking: {
+          date: baseSlot.date,
+          interval: Object.keys(baseSlot.intervals)[0],
+          bookingNotes: "This is a booking note",
+        },
+        initialAttendance: {
+          bookedInterval: Object.keys(baseSlot.intervals)[0],
+          attendedInterval: Object.keys(baseSlot.intervals)[0],
+          bookingNotes: "This is a booking note",
+        },
+        update: {
+          date: baseSlot.date,
+          interval: Object.keys(baseSlot.intervals)[0],
+        },
+        wantAttendance: {
+          bookedInterval: Object.keys(baseSlot.intervals)[0],
+          attendedInterval: Object.keys(baseSlot.intervals)[0],
+        },
+      },
+
+      {
+        name: "delete booking",
+        initialBooking: {
+          date: baseSlot.date,
+          interval: Object.keys(baseSlot.intervals)[0],
+        },
+        initialAttendance: {
+          bookedInterval: Object.keys(baseSlot.intervals)[0],
+          attendedInterval: Object.keys(baseSlot.intervals)[0],
+        },
+        update: null, // null = delete the slot!
+        wantAttendance: null,
+      },
+    ]);
+
+    // test.skip("create booking: interval with notes", async () => {
```

Comment:
I'm curious: why commenting out a skipped test? I'd say it should be either be left alone or removed, but not commented out. I'm probably missing a possible reason to do this.

Reply:
Thanks, this is a (forgotten) leftover. Will fix with in the currently open PR when resolving conflicts